src/components/imageContainer/ImageContainer.st.css

```diff
@@ -0,0 +1,18 @@
+.mainImg {
+  height: 100%;
+  width: 50em;
+  background-size: cover;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.rightArrow{
+  transform: rotate(180deg);
+}
+
+.arrow{
```

Comment:
so i guess this is a left arrow?

Reply:
its a general arrow.
right arrow is extension of arrow